djstripe/models/billing.py

```diff
@@ -1322,15 +1392,50 @@ class Subscription(StripeModel):
         help_text="The quantity applied to this subscription. This value will be "
         "`null` for multi-plan subscriptions",
     )
-    start = StripeDateTimeField(
+    # TODO: schedule (implement model SubscriptionSchedule, see #899)
+    legacy_start = StripeDateTimeField(
         help_text="Date of the last substantial change to "
         "this subscription. For example, a change to the items array, or a change "
         "of status, will reset this timestamp."
     )
+    start_date = StripeDateTimeField(
+        null=True,
+        blank=True,
+        help_text="Date when the subscription was first created. The date "
+        "might differ from the created date due to backdating.",
+    )
     status = StripeEnumField(
         enum=enums.SubscriptionStatus, help_text="The status of this subscription."
     )
-    tax_percent = StripePercentField(
+
+    @property
+    def billing(self):
+        warnings.warn(
+            "Invoice.billing has been renamed to .collection_method. "
+            "This alias will be removed in djstripe 2.4",
+            DeprecationWarning,
+        )
+        return self.collection_method
+
+    @property
+    def start(self):
+        warnings.warn(
+            "Invoice.start is deprecated and this alias will be removed in "
+            "djstripe 2.4. USe Invoice.start_date instead.",
+            DeprecationWarning,
+        )
+        return self.legacy_start
+
+    @property
+    def tax_percent(self):
+        warnings.warn(
+            "Subscription.tax_percent has been deprecated and will be removed "
+            "in djstripe 2.4",
```

Comment:
Mention that default_tax_rates should be used instead

Reply:
:+1: